pkg/utils/placements.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func GetTolerationForCSIPods() corev1.Toleration {
+
+	runOnMaster := true
+	var err error
+	rom := os.Getenv(runCSIDaemonsetOnMaster)
+	if rom != "" {
+		runOnMaster, err = strconv.ParseBool(rom)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+	if runOnMaster {
+		toleration := corev1.Toleration{
+			Key:      "node-role.kubernetes.io/master",
+			Operator: corev1.TolerationOpEqual,
+			Value:    "true",
```

Comment:
value is not required as we will not have it set for the master node and also the Operator should be exists not equal. please check https://docs.openshift.com/container-platform/4.14/nodes/scheduling/nodes-scheduler-taints-tolerations.html